micrometer-core/src/main/java/io/micrometer/core/instrument/Observer.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micrometer.core.instrument;
+
+/**
+ * @author Dmitry Poluyanov
+ * @since 23.07.17
+ */
+public interface Observer {
```

Comment:
FWIW, [this thread](https://groups.google.com/forum/#!topic/netflix-atlas/iUtiMEGLZJo) has some details on what we are currently using at Netflix and the error compared to t-digests. The current histogram approach we use in spectator has worked fairly well for us over the last 2 years. The main draw back and where we are looking to improve it is to reduce the number of buckets needed, and thus are storage and query costs, to get acceptable accuracy.